source/documentationUtils.py

```diff
@@ -47,7 +50,7 @@ def getDocFilePath(fileName: str, localized: bool = True) -> Optional[str]:
 		return None
 	else:
 		# Not localized.
-		if NVDAState.isRunningAsSource() and fileName in ("copying.txt", "contributors.txt"):
+		if NVDAState.isRunningAsSource() and fileName in ("copying.txt"):
```

Comment:
What is between parenthesis is not a tuple anymore!

```suggestion
		if NVDAState.isRunningAsSource() and fileName == "copying.txt":
```